lib/task.js

```diff
@@ -105,6 +105,28 @@ export async function loadTasksForJob( jobUri ){
   return tasks;
 }
 
+export async function isTaskExist(graph, job, index, operation) {
+    const queryStr = `
+        ${PREFIXES}
+        SELECT ?task {
+            GRAPH <${graph}> {
+                VALUES ?status {
+                    <${STATUS_SCHEDULED}>
```

Comment:
might be better to just check for status != failed ?

Reply:
changed